packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/GitHub/GithubLogin.tsx

```diff
@@ -35,8 +37,8 @@ export const GithubLogin = () => {
             <Icon name="info" size={12} />
           </Element>
           <Text lineHeight="16px" size={12}>
-            CodeSandbox needs access to your public repositories in order to
-            link and/or export a public sandbox.
+            CodeSandbox needs access to your {privacyLabel} repositories in
+            order to link and/or export a {privacyLabel} sandbox.
```

Comment:
Should we take `unlisted` into account to?

Reply:
@tristandubbeld we should! [14064fc](https://github.com/codesandbox/codesandbox-client/pull/7426/commits/14064fc3fc50f622dd2b2299129b3b7f5e955c61)